lib/Backends/Interpreter/tests/InterpreterOperatorTest.cpp

```diff
@@ -23,4 +23,5 @@ std::set<std::string> glow::backendTestBlacklist = {
     "RepeatedSLSWithPartialTensors/0",
     "SigmoidSweep_Float16/0",
     "TanHSweep_Float16/0",
+    "TopK1int32/0",
```

Comment:
In general we strive to keep support for everything in the Interpreter -- can we add support for this? Hopefully simple as templatizing the kernel in `InterpreterNodes.cpp`?